.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,18 @@
+*   Version of collectd:
+*   Operating system / distribution:
+
+## Expected behavior
+
+(Description of the behavior / output that you expected)
+
+## Actual behavior
+
+(Description of the behavior / output that you observed)
+
+## Steps to reproduce
+
+*   step 1
+*   step 2
+*   step 3
+
+<!-- vim: set syntax=markdown : -->
```

Comment:
I copied the file name from the doc verbatim. Upon close reading, it turns out that they don't care about case and ignore filename extensions, so I've moved the file to `issue_template.md`.